test/tracing/reactnavigationv4.test.ts

```diff
@@ -498,6 +501,7 @@ describe('ReactNavigationV4Instrumentation', () => {
       if (confirmedContext) {
         expect(confirmedContext.name).toBe(route2.routeName);
         expect(confirmedContext.data).toBeDefined();
+        expect(confirmedContext.metadata).toBeUndefined();
```

Comment:
I'd remove this line and the others but the `onRouteConfirmed`.

Reply:
This one is actually on `onRouteConfirmed` too, just the GitHub UI makes it look like it is in `transaction not attached on a cancelled navigation`.